packages/main/src/SegmentedButtonItem.ts

```diff
@@ -0,0 +1,110 @@
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+
+import type { I18nText } from "@ui5/webcomponents-base/dist/i18nBundle.js";
```

Comment:
this type will be redundant when you rebase. See #6366

Reply:
removed